definition.go

```diff
@@ -1285,9 +1286,26 @@ func (gl *NonNull) Error() error {
 
 var NameRegExp, _ = regexp.Compile("^[_a-zA-Z][_a-zA-Z0-9]*$")
```

Comment:
Yes and no. It's not used in the lib anymore, but it's exposed and part of the public API therefore I left it in because I didn't want to break backwards compatibility.